src/git.ts

```diff
@@ -107,3 +109,20 @@ export async function fetch(token: string | undefined, branch: string, ...option
 
     return cmd(...args);
 }
+
+export async function clone(
+    token: string | undefined,
+    ghRepository: string,
+    branch: string,
+    ...options: string[]
+): Promise<string> {
+    core.debug(`Executing 'git clone' to branch '${branch}' with token and options '${options.join(' ')}'`);
+
+    const remote = token !== undefined ? getRemoteUrl(token, ghRepository) : 'origin';
```

Comment:
it is not clear to me what is the expected value for `ghRepository`. Should it be the full url to the other repository? If so then I think that `getRemoteUrl` will append `/${owner}/${repo}.git` to that url which doesn't seem like a desired effect

Reply:
I can't tell what the original author intended, but I would think ghRepository would be the full URL to the target repository.